userspace/engine/yaml_helper.h

```diff
@@ -85,6 +85,12 @@ class yaml_helper {
 	inline static const std::string validation_failed = "failed";
 	inline static const std::string validation_none = "none";
 
+	enum config_files_strategy {
+		STRATEGY_APPEND,  // append to existing sequence keys, override scalar keys and add new ones
+		STRATEGY_OVERRIDE,  // override existing keys (sequences too) and add new ones
+		STRATEGY_ADDONLY    // only add new keys and ignore existing ones
+	};
+
```

Comment:
It might be overkill but shouldn't we have something here that allows to convert this enum from/to string ?
This would keep all the conversions in one place close to the enum.

Reply:
> It might be overkill but shouldn't we have something here that allows to convert this enum from/to string

Yep i agree! Will do that in a moment!

Reply:
Done!